js/widgets/toolbar.js

```diff
@@ -60,15 +60,13 @@ define( [
 				var currentTheme = this.options.theme ? this.options.theme : "inherit",
 					newTheme = o.theme ? o.theme : "inherit";
 
-				this.element.removeClass( "ui-bar-" + currentTheme ).addClass( "ui-bar-" + newTheme );
+				this._removeClass( null, "ui-bar-" + currentTheme );
```

Comment:
@arschmitz can you explain what that means, i thought that is where i was putting this... or is it that since i'm using extra do I need to explicitly add this.element like `_removeClass( this.element, null, "ui-bar-" + currentTheme );`